### ISSUE TITLE
UI: Add API url in API Key window

### ISSUE THREAD

**Issue by pcrespov** [ITISFoundation/osparc-simcore#2280]
When a user generate the API keys, it would be handy to have the base-url as well at hand (in all deploys except for AWS-production osparc.io because that is the default). Those are the three elements needed to setup the API configuration

![image.png](https://images.zenhubusercontent.com/5caef818ecad11531cc41364/418eb50d-8132-4aea-8f6c-45b3b4c95160)
```[tasklist]
### Event Horizon
- [ ] https://github.com/ITISFoundation/osparc-simcore/pull/6843
- [ ] https://github.com/ITISFoundation/osparc-simcore/issues/6340
```

**Comment by pcrespov** [comment]
- add api url in statics so that front-end can display correct. 
- add doc api as well

**Comment by pcrespov** [comment]
@giancarloromeo please return the api entrypoint in the response to create an api-key. Check `app.request.url`

**Comment by giancarloromeo** [comment]
See https://github.com/ITISFoundation/osparc-simcore/issues/6340 for proper `api_base_url` generation.

**Comment by giancarloromeo** [comment]
@odeimaiz the updated OpenAPI Spec is in `services\web\server\src\simcore_service_webserver\api]v0\openapi.yaml`

![Image](https://github.com/user-attachments/assets/b0a4586c-dc18-4128-a924-477bf1649ddd)

The API Key creation returns (just the first time!) 2 new fields:
- `id` will be used for subsequent `GET` and `DELETE` operations
- `api_base_url` must be returned in the UI as sketched above.

**Comment by giancarloromeo** [comment]
Addressed in the other issue #6843
